boost.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "boost",
+    "buildsystem": "simple",
+    "build-commands": [
+        "./bootstrap.sh --prefix=/app --with-libraries=system",
+        "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
```

Comment:
1.78.0 is out

Reply:
I'll update and test. Side note, should this be changed in the master manifest repo too?
https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/boost.json#L11

Reply:
Yes, we need to update it there too. I forgot to do it :).